<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ColorForm</class>
 <widget class="QDialog" name="ColorForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>300</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>300</height>
   </size>
  </property>
  <property name="sizeIncrement">
   <size>
    <width>400</width>
    <height>300</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>400</width>
    <height>300</height>
   </size>
  </property>
  <property name="font">
   <font>
    <pointsize>15</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="2" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QPushButton" name="pBut7_cyan">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
     background-color: cyan;
	color: rgb(0, 0, 0);
}</string>
       </property>
       <property name="text">
        <string>Cyan</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pBut8_magenta">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
     background-color: magenta;
}</string>
       </property>
       <property name="text">
        <string>Magenta</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pBut9_yellow">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
     background-color: yellow;
	color: rgb(0, 0, 0);
}</string>
       </property>
       <property name="text">
        <string>Yellow</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="pBut4_red">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
     background-color: red;
}</string>
       </property>
       <property name="text">
        <string>Red</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pBut5_green">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
     background-color: green;
}</string>
       </property>
       <property name="text">
        <string>Green</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pBut6_blue">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
     background-color: blue;
}</string>
       </property>
       <property name="text">
        <string>Blue</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_1">
     <item>
      <widget class="QPushButton" name="pBut1_white">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
     background-color: white;
	color: rgb(0, 0, 0);
}

 QPushButton:pressed {
     background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                       stop: 0 #dadbde, stop: 1 #37B6CE);
 }</string>
       </property>
       <property name="text">
        <string>White</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pBut2_black">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
     background-color: black;
}</string>
       </property>
       <property name="text">
        <string>Black</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pBut3_gray">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
     background-color: gray;
}</string>
       </property>
       <property name="text">
        <string>Gray</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
